frontend/apps/reader/modules/readerstyletweak.lua

```diff
@@ -875,7 +875,7 @@ function ReaderStyleTweak:editBookTweak(touchmenu_instance)
                 -- being shown: having them identical will hide that.
             end
         end,
-        close_discarded_notif_text = NOT_MODIFIED_MSG;
+        close_discarded_notif_text = NOT_MODIFIED_MSG
```

Comment:
This one is the last table element. Best to replace it with a comma so this line is later not modified when we add an element.

Reply:
I was looking for the KOReader code styling guide right now. But I did not find it :/
Just for documentation: Do you have a link at hand.

Reply:
https://github.com/koreader/koreader-base/wiki/Coding-style
(It's linked on the right column of the Wiki under Development - not sure why this page is in the wiki of koreader-base - I had never visited https://github.com/koreader/koreader-base/wiki before :)

Reply:
Maybe this should be linked in http://koreader.rocks/doc/topics/Coding-style? Dunno.